.github/workflows/c3-e2e-quarantine.yml

```diff
@@ -12,6 +12,7 @@ env:
   bun-version: 1.0.3
 jobs:
   e2e:
+    timeout-minutes: 30
     name: ${{ format('E2E ({0})', matrix.pm) }}
```

Comment:
I notice this file doesn't have an `if: github.repository_owner == 'cloudflare' && ...` in it - could we add one? It seems to consistently run & fail on forks

Reply:
Yes please!